django/website/logframe/migrations/0004_auto_20151215_1203.py

```diff
@@ -10,9 +10,9 @@ def set_logframe_slug(apps, schema_editor):
     LogFrame = apps.get_model('logframe', 'LogFrame')
     for logframe in LogFrame.objects.all():
         slug = logframe.name.lower()
-        slug.replace(' ', '_')
+        slug = slug.replace(' ', '_')
         slug = re.sub('[^\w-]', '', slug)
-        slug = slug[:47]
+        slug = slug[:46]
         if LogFrame.objects.filter(slug=slug).exists():
             count = LogFrame.objects.filter(slug__startswith=slug).count() + 1
             slug += unicode(count)
```

Comment:
This change does two things. The first is fixing a bug in the migration where by spaces were never removed from the slug, and the second is to allow for the number on the end of logframes to run into three digits. This makes it very unlikely that people will run out of valid slugs for a particular name.